crypto/src/conversation/decrypt.rs

```diff
@@ -799,5 +807,36 @@ pub mod tests {
             )
             .await
         }
+
+        #[apply(all_credential_types)]
+        #[wasm_bindgen_test]
+        pub async fn returns_sender_client_id(credential: CredentialSupplier) {
```

Comment:
Is it possible to add a test for when it won't be present? Do we know which cases it will be none?